baselines/asset/src/v1/asset_service_client.ts.baseline

```diff
@@ -115,6 +115,12 @@ export class AssetServiceClient {
     // Save the auth object to the client, for use by other methods.
     this.auth = (this._gaxGrpc.auth as gax.GoogleAuth);
 
+    // Set useJWTAccessAlways on the auth object.
+    this.auth.useJWTAccessAlways = true;
```

Comment:
Just to confirm: there is no opt out, it will be always set to `true`, the user cannot override this in any way. Is it how it's supposed to be?

Reply:
I can't think of a reason why a user would want to opt out. This variable is here to allow us to differentiate a GAPIC library from another library, e.g., sheets, as only the GAPIC libraries work with self signed JWT.

We should be mindful of testing this with DIREGAPIC libraries, now that I think of it 🤔

Reply:
As long as this variable can be set to false for DIREGAPIC, then this should be sufficient

Reply:
We've added the DIREGAPIC variable now.